src/internal/model/model.go

```diff
@@ -136,6 +107,15 @@ func (o *Observation) BoolValue() (bool, error) {
 	return strconv.ParseBool(o.Value)
 }
 
+// StringValue returns a string representation of the Observation value.
+func (o *Observation) StringValue() (string, error) {
```

Comment:
I think we can have something like `JSONValue` instead, which is going to accept a value that can be passed further to `json.Unmarshal`